docs/guides/route_filtering/inbound/default_route.md

```diff
@@ -15,3 +15,17 @@ On the other hand, if you want the full routing table, you should not accept any
     ip prefix-list default-route permit 0.0.0.0/0
     ipv6 prefix-list default-route permit ::/0
     ```
+
+    Route-Map statement for allowing *only* default routes in:
+    ```
+    route-map prefixes-in permit 10
+      match ip address prefix-list default-route
```

Comment:
Hm, IIRC IPv4 and IPv6 prefix lists need to have different names, at least on some IOSes :thinking: Hence I'd propose to go for `default-v4` and `default-v6` here (and above).

Reply:
renamed them. remark: this config came from a real router, so at least for one version of IOS it worked.